types/react-navigation/index.d.ts

```diff
@@ -702,6 +702,7 @@ export function TabNavigator(
 export interface TabBarTopProps {
   activeTintColor: string;
   inactiveTintColor: string;
+  indicatorStyle: ViewStyle;
```

Comment:
@timwangdev Shouldn't this be `StyleProp<ViewStyle>`?

Reply:
@iRoachie updated.

Reply:
@iRoachie Should be `StyleProp<ViewStyle>`, but I never tested by using `RegisteredStyle`. Seems we got this wrong on other style props in this file.